backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/ConstraintUsageImpl.java

```diff
@@ -431,4 +433,15 @@ public Object eInvoke(int operationID, EList<?> arguments) throws InvocationTarg
         return super.eInvoke(operationID, arguments);
     }
 
+    /**
+     * @generated NOT
+     */
+    @Override
+    public Feature namingFeature() {
+        OwningMembership owMembership = this.getOwningMembership();
+        if (owMembership instanceof RequirementConstraintMembership reqMembership && this.getOwnedReferenceSubsetting() != null) {
```

Comment:
You declare `reqMembership` but you never use it. could you fix that please?

Reply:
Done.